pontoon/base/templates/base.html

```diff
@@ -29,6 +29,10 @@
       {% endblock %}
     {% endblock %}
 
+    <!-- Add Chart.js and Moment CDN link -->
+    <script src="https://cdn.jsdelivr.net/npm/chart.js@4.4.2/dist/chart.umd.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/moment@^2.30.1"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chartjs-adapter-moment@^1.0.1"></script>
```

Comment:
We try to avoid loading static files from external sources. Please instead include the files in the repo.